docs-src/deployments/verification_frogs.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: layouts/deployment.11ty.js
+title: Frog Verification
+---
+
+<oe-verification-grid id="verification-grid" grid-size="1">
+    <oe-verification verified="true" shortcut="Q"></oe-verification>
+    <oe-verification verified="false" shortcut="W"></oe-verification>
+    <oe-verification verified="true-multiple" shortcut="E"></oe-verification>
+    <oe-verification verified="false-multiple" shortcut="R"></oe-verification>
```

Comment:
`verified` accepts a limited number of values defined here:

https://github.com/ecoacoustics/web-components/blob/main/src/models/decisions/decision.ts#L5-L10

Possible values:

- `false`
- `true`
- `unsure`
- `skip`

We did this so that the `oe-verified` column has predictable values that can be ingested by anything that conforms to the annotation format.

What you probably want is `additional-tags="multiple"`

If you use `additional-tags="multiple"`, you'll have the `oe-verified` column in the downloaded results, but also have an additional `oe-multiple` column which will either be `true` or empty.

Reply:
```suggestion
    <oe-verification verified="true" additional-tags="multiple" shortcut="E"></oe-verification>
    <oe-verification verified="false" additional-tags="multiple" shortcut="R"></oe-verification>
```

Reply:
If you have the permissions make the change for @Slade-AA

Reply:
For context, this is what the additional-tags solution would look like

![image](https://github.com/user-attachments/assets/42dc30da-9093-4e83-8f10-6f199431ed28)

Reply:
Yeah that seems to be what he wants